xml/System.IO/FileSystemInfo.xml

```diff
@@ -1318,9 +1318,8 @@ The `Extension` property returns the <xref:System.IO.FileSystemInfo> extension,
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>If this <see cref="T:System.IO.FileSystemInfo" /> instance represents a link, returns the link target's path.
-            If a link does not exist in <see cref="P:System.IO.FileSystemInfo.FullName" />, or this instance does not represent a link, returns <see langword="null" />.</summary>
-        <value>To be added.</value>
+        <summary>Gets the target path of the symbolic link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</summary>
```

Comment:
```suggestion
        <summary>Gets the target path of the link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</summary>
```
Do not specify symbolic link as this API also works with junctions which are another kind of link only supported by NTFS.

Reply:
Noted @Jozkee. Made the change